### PR TITLE
Write headers in country csv

### PR DESCRIPTION
**Pull request by grischard**

### CODE REVIEW

File: tilelog/country.py [hunk 1]
```diff
@@ -24,4 +24,8 @@ def country_logs(curs, date, dest):
                   "min_requests": tilelog.constants.MIN_TILE_REQUESTS})
     csvwriter = csv.writer(dest, dialect=csv.unix_dialect,
                            quoting=csv.QUOTE_NONNUMERIC)
+    # Write the header row
+    csvwriter.writerow(["country", "ips", "tps", "tps_miss"])
```
Comment (pnorman) [inline tilelog/country.py:28]: Is there any reason [writeheader](https://docs.python.org/3/library/csv.html#csv.DictWriter.writeheader) won't work?

**Review by pnorman — CHANGES_REQUESTED** [review]
What's the plan for existing files, if any, as this means we'll have published files with two formats

**Comment by grischard** [comment]
Good point about existing files! We can add the header and append the existing content, then use sponge from [moreutils](https://rentes.github.io/unix/utilities/2015/07/27/moreutils-package/) to overwrite the original file

```bash
#!/bin/bash

# robustness
set -euo pipefail
# Verify that at least one file exists before proceeding for each type
shopt -s nullglob

sudo apt install moreutils

# Function to add headers only if they don't exist
add_header_if_not_exists() {
  local filename=$1
  local header=$2

  # Check if header already exists
  if ! head -n 1 "$filename" | grep -q "^${header}$"; then
    { echo "$header"; cat "$filename"; } | sponge "$filename"
  fi
}

# Add headers to hosts-* files
hosts_files=(hosts-*.csv)
if [ "${#hosts_files[@]}" -gt 0 ]; then
  for filename in "${hosts_files[@]}"; do
  add_header_if_not_exists "$filename" "\"host\",\"tps\",\"tps_miss\""
  done
fi

# Add headers to apps-* files
apps_files=(apps-*.csv)
if [ "${#apps_files[@]}" -gt 0 ]; then
  for filename in "${apps_files[@]}"; do
    add_header_if_not_exists "$filename" "\"app\",\"tps\",\"tps_miss\""
  done
fi

# Add headers to country-* files
country_files=(country-*.csv)
if [ "${#country_files[@]}" -gt 0 ]; then
  for filename in "${country_files[@]}"; do
    add_header_if_not_exists "$filename" "\"country\",\"ips\",\"tps\",\"tps_miss\""
  done
fi
```
I don't have access to that server, though.

**Comment by grischard** [comment]
@pnorman I've added headers to the other files.

File: tilelog/host.py [hunk 2]
```diff
@@ -67,6 +67,10 @@ def host_logs(curs, date, dest):
                            if metrics[0] >= tilelog.constants.MIN_TPS],
                           key=lambda host: host[1], reverse=True)  # Sort by TPS
     click.echo("Writing host usage to file")
+
```
Comment (pnorman) [inline tilelog/host.py]: Stray whitespace insertion

File: tilelog/country.py [hunk 3]
```diff
@@ -24,4 +24,8 @@ def country_logs(curs, date, dest):
...
+
```
Comment (pnorman) [inline tilelog/country.py]: Stray whitespace that's not present in the other changes

**Review by pnorman — CHANGES_REQUESTED** [review]
Looks good as per discussion, but I need to test and there's a couple of whitespace changes

**Review by pnorman — APPROVED** [review]
